src/functions-and-arrays.js

```diff
@@ -28,9 +118,43 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
+function uniquifyArray (wordsArr) {
+  // test empty array
+  if (wordsArr.length === 0) {
+    return null;
+  }
+  // procedure
+  for (let i = 0; i < wordsArr.length-1; i++) {
+    let check = wordsArr.indexOf(wordsArr[i], i+1);
+    // only starts the procedure if there is a match
```

Comment:
Estas seguro que queres anidar for loops?
Esto incremente exponencialmente el tiempo de ejecucion, creo que hay soluciones que no requieren este grado de complejidad

Reply:
Entiendo, tenía el problema cuando había alguna palabra repetida en múltiples ocasiones y esa fue la única solución que se me ocurrió.

Reply:
pero no podes hacer algo asi?
```

let uniqueArray = []
if(wordsArray.indexOf(array[i]) === i) {
  uniqueArray.push(array[i])
}

return uniqueArray
```

Reply:
Vale, me ha costado un poco entenderla pero ya veo que es mucho más sencillo. No creo que se me hubiera ocurrido. Cojo la idea!